File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/metadata/delete/MetadataDeleteRequestTest.java
package uk.ac.jisc.rdss.tests.metadata.delete;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class MetadataDeleteRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/metadata/delete/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/metadata/delete/request.json");
    }
}

File: message-api-schema-validator/tests/metadata/metadata_delete_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class MetadataDeleteRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return "messages/metadata/delete/request_schema.json"

    def runTest(self):
        self.validate_json("messages/metadata/delete/request.json")
